tests/test_smoke.py

```diff
@@ -2214,6 +2214,20 @@ def tmp_local_list_storage_obj(self, tmp_bucket_name, tmp_source):
         yield from self.yield_storage_object(name=tmp_bucket_name,
                                              source=list_source)
 
+    @pytest.fixture
+    def tmp_bulk_del_storage_obj(self, tmp_bucket_name, tmp_source):
+        # Creates a temporary storage object for testing bulk deletion.
+        # Stores must be added in the test.
+        with tempfile.TemporaryDirectory() as tmpdir:
+            subprocess.check_output(f'mkdir -p {tmpdir}/folder{{000..255}}',
+                                    shell=True)
+            subprocess.check_output(f'touch {tmpdir}/test{{000..255}}.txt',
+                                    shell=True)
+            subprocess.check_output(
+                f'touch {tmpdir}/folder{{000..255}}/test.txt', shell=True)
+            yield from self.yield_storage_object(name=tmp_bucket_name,
+                                                 source=tmp_source)
```

Comment:
```suggestion
    def tmp_bulk_del_storage_obj(self, tmp_bucket_name):
        # Creates a temporary storage object for testing bulk deletion.
        # Stores must be added in the test.
        with tempfile.TemporaryDirectory() as tmpdir:
            subprocess.check_output(f'mkdir -p {tmpdir}/folder{{000..255}}',
                                    shell=True)
            subprocess.check_output(f'touch {tmpdir}/test{{000..255}}.txt',
                                    shell=True)
            subprocess.check_output(
                f'touch {tmpdir}/folder{{000..255}}/test.txt', shell=True)
            yield from self.yield_storage_object(name=tmp_bucket_name,
                                                 source=tmpdir)
```
This one needs to yield storage object using directory with bulk as a `source`.